support/routes/v2/isobot/index.ts

```diff
@@ -1,63 +1,47 @@
-import express, { RequestHandler } from "express"
+import { App, ExpressReceiver } from "@slack/bolt"
 
 import { Whitelist } from "@database/models"
+import config from "@root/config/config"
 import logger from "@root/logger/logger"
 import WhitelistService from "@root/services/identity/WhitelistService"
-import type { DnsCheckerResponse } from "@root/types/dnsChecker"
 
-import BotService, { SlackPayload } from "./ops/botService"
+import BotService from "./ops/botService"
 
-export const isobotRouter = express.Router()
 const botService = new BotService(
   new WhitelistService({ repository: Whitelist })
 )
+const signingSecret = config.get("slackbot.secret")
+const token = config.get("slackbot.token")
 
-const handleWhitelistEmails: RequestHandler<
-  {},
-  { message: string },
-  SlackPayload,
-  {},
-  {}
-> = async (req, res) => {
-  const slackTimestamp = req.headers["x-slack-request-timestamp"] as string
-  const slackSig = req.headers["x-slack-signature"] as string
-
-  if (!slackTimestamp || !slackSig)
-    return res.send({ message: "Missing header/signature" })
```

Comment:
we don't need this as `bolt` provides this by default - see [here](https://api.slack.com/authentication/verifying-requests-from-slack#SDK-support)